pycode/memilio-simulation/memilio/simulation/abm.cpp

```diff
@@ -150,24 +149,31 @@ PYBIND11_MODULE(_simulation_abm, m)
         .def_property_readonly("age", &mio::abm::Person::get_age)
         .def_property_readonly("is_in_quarantine", &mio::abm::Person::is_in_quarantine);
 
+    py::class_<mio::abm::TestingCriteria>(m, "TestingCriteria")
+        .def(py::init<const std::vector<mio::abm::AgeGroup>&, const std::vector<mio::abm::LocationType>&, const std::vector<mio::abm::InfectionState>&>(), py::arg("age_groups"), py::arg("location_types"), py::arg("infection_states"));
+    
+    py::class_<mio::abm::GenericTest>(m, "GenericTest")
+        .def(py::init<>());
+    py::class_<mio::abm::AntigenTest, mio::abm::GenericTest>(m, "AntigenTest")
+        .def(py::init<>());
+    py::class_<mio::abm::PCRTest, mio::abm::GenericTest>(m, "PCRTest")
+        .def(py::init<>());
+    
     py::class_<mio::abm::TestingScheme>(m, "TestingScheme")
-        .def(py::init<mio::abm::TimeSpan, double>(), py::arg("interval"), py::arg("probability"))
-        .def_property("interval", &mio::abm::TestingScheme::get_interval, &mio::abm::TestingScheme::set_interval)
-        .def_property("probability", &mio::abm::TestingScheme::get_probability,
-                      &mio::abm::TestingScheme::set_probability);
-
+        .def(py::init<const std::vector<mio::abm::TestingCriteria>&, mio::abm::TimeSpan, mio::abm::TimePoint, mio::abm::TimePoint, mio::abm::GenericTest, double>(), py::arg("testing_criteria"), py::arg("testing_min_time_since_last_test"), py::arg("start_date"), py::arg("end_date"), py::arg("test_type"), py::arg("probability"))
```

Comment:
```suggestion
        .def(py::init<const std::vector<mio::abm::TestingCriteria>&, mio::abm::TimeSpan, mio::abm::TimePoint, mio::abm::TimePoint, const mio::abm::GenericTest&, double>(), py::arg("testing_criteria"), py::arg("testing_min_time_since_last_test"), py::arg("start_date"), py::arg("end_date"), py::arg("test_type"), py::arg("probability"))
```

I haven't worked with pybind for a while but why not const& on GenericTest here?

Reply:
Simply forgot it, fixed it.